README.md

```diff
@@ -115,6 +115,12 @@ We are very happy to get code contributions and features from the global LORIS c
 
 If you would like to contribute to LORIS development, please consult our [Contributing Guide](./CONTRIBUTING.md).
 
+Code is merged after a given pull request is reviewed using our code review checklist, has been approved by 2 reviewers, and is tagged [PassedManualTests](https://github.com/aces/Loris/labels/Passed%20Manual%20Tests). The [new Pull Request template](https://github.com/aces/Loris/blob/main/.github/PULL_REQUEST_TEMPLATE.md) has guidelines for how to submit a contribution.
+
+If there are any issues with your pull request, please address them and assign it back to the original reviewer.
+
+Please also visit [the contributing guide](https://github.com/aces/Loris/blob/main/CONTRIBUTING.md) for more details.
```

Comment:
this text seems to belong in the Contribution Guide which is linked to (properly) one line above.

You are also linking to it a second time... why ?